hack/run-functests.sh

```diff
@@ -69,7 +69,9 @@ fi
 if [ ! -f "cnf-tests/bin/junit-merger" ]; then
   go build -o cnf-tests/bin/junit-merger cnf-tests/testsuites/pkg/junit-merger/junit-merger.go
 fi
-
+if [ ! -f "cnf-tests/bin/j2html" ] && [ "$JUNIT_TO_HTML" == "true" ]; then
+    go build -o cnf-tests/bin/j2html cnf-tests/testsuites/pkg/j2html/j2html.go
+fi
```

Comment:
I mean moving this below line 144

Reply:
I think it's better to keep all building code in one place consolidated

Reply:
Ok. I agree that this choice will benefit more in improving readability then saving us from building this bin which is small anyway.

Reply:
it's lgtm too.